README.md

```diff
@@ -165,150 +162,126 @@ upper right corner.  ![](https://raw.githubusercontent.com/deephaven-examples/de
 ![](https://raw.githubusercontent.com/deephaven-examples/deephaven-ib/main/docs/assets/allow-connections.png)
 
+## Virtual Environment
 
-## Launch
-
-There are multiple ways to launch [Deephaven](https://deephaven.io) with [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) 
-installed.  The launch can either happen via a local installation or via Docker images.
-
-For a Docker Compose example, see [docker/release/](docker/release/).
+Interactive Brokers does not make their Python wheels available via PyPI, and the wheels are not redistributable.
+As a result, installing [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) requires a script to build the wheels locally before installation.
 
+To keep your development environment clean, the script creates a virtual environment for [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib).
 
-### (Option 1) Launch pip-installed Deephaven with Docker -- interactive
+### Build the Virtual Environment
 
-The pip-installed Deephaven uses a lightweight Deephaven installation that is installed using pip.  In this case,
-the pip-installed Deephaven system is installed in a Docker container.
-
-1) Create a directory for your data and scripts
+1) Install Java 17 and set the appropriate `JAVA_HOME` environment variable.    
+2) Check out [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib)
     ```bash
-    mkdir data
+    git clone git@github.com:deephaven-examples/deephaven-ib.git
     ```
-2) Create a directory for your Deephaven IDE configuration and notebooks
-   ```bash
-   mkdir `pwd`/.deephaven
-   ```
-3) Launch the system:
+3) Change to the deephaven-ib directory:
     ```bash
-    # Set jvm_args to the desired JVM memory for Deephaven
-    docker run -it -v data:/data -v `pwd`/.deephaven:/storage -p 10000:10000 ghcr.io/deephaven-examples/deephaven-ib python3 -i -c "from deephaven_server import Server; _server = Server(port=10000, jvm_args=['-Xmx4g','-Dauthentication.psk=DeephavenRocks!']); _server.start()"
+    cd deephaven-ib
     ```
-4) Launch the [Deephaven IDE](https://github.com/deephaven/deephaven-core/blob/main/README.md#run-deephaven-ide) by navigating to [http://localhost:10000/ide/](http://localhost:10000/ide/) in a browser and logging in with the password `DeephavenRocks!`.
-
-
-### (Option 2) Launch pip-installed Deephaven with Docker -- run a script
+4) Build a [deephaven-ib](https://github.com/deephaven-examples/deephaven-ib) virtual environment:
 
-The pip-installed Deephaven uses a lightweight Deephaven installation that is installed using pip.  In this case,
-the pip-installed Deephaven system is installed in a Docker container.  
-This is a good option for production scenarios where scripts need to be run and related data needs to be visualized.
-
-1) Create a directory for your data and scripts
+   First, install the dependencies needed to run the script:
```

Comment:
Maybe the script should create an empty tmp venv, install dependencies there using the command below, use that to create the dhib_env, then clean up the tmp venv. Might not be the best way to do it. I immediately feel chicken-and-egg when asked to install some Python packages to build another venv.

Reply:
If someone wants to create a venv instead of directly installing those packages, they can certainly do it.  It doesn't make sense to do it here.  That is completely up to them on how they want to manage it.